### ISSUE TITLE
[Review recommendation] Add a .janno colum to document the broad chronological era of a sample

### ISSUE THREAD

**Issue by nevrome** [poseidon-framework/poseidon-schema#83]
This recommendation was raised in the [review of the Poseidon paper](https://elifesciences.org/reviewed-preprints/98317v1/reviews#tab-content#tab-content).

**Comment by nevrome** [comment]
Broad chronological era or even cultural attributions would be useful but are hard to maintain as fixed-choice fields given the heterogeneity of terms on a worldwide scale and the many unresolved controversies around their precise definition.

I think this leads to a foundational decision we have to make for Poseidon: Do we want to include more archaeological information with all it's uncertainties or draw a line where we currently are?

**Comment by stschiff** [comment]
I share your skepticism. One thing we could potentially do is to add a column for a URI identifying a specific culture or time period, which could resolve to a Wikidata entry, or some other entity. But we could also simply push back on this one.

**Comment by nevrome** [comment]
Maybe this could indeed be the moment to start integrating with the linked open data world. We could create a column that expects a URI, for example a period.do permalink: e.g. http://n2t.net/ark:/99152/p0zj6g8ks9s

The biggest challenge here is imho to provide a documented workflow on how to find the most relevant entity for the package creator.

And for the validation we would need an (optional, because extremely expensive?) process to resolve the URIs and make sure that the referenced entities have certain minimal properties.

**Comment by stschiff** [comment]
Yes, you've summarized the challenges. Needs to be discussed further when we deal with the revision.

**Comment by stschiff** [comment]
I will check with the MixS MINAS and make a proposal

**Comment by nevrome** [comment]
I'm closing issues that are already implemented in #93.
